cli/cli.go

```diff
@@ -393,13 +393,61 @@ func getCliContext() *cli.App {
 	}
 	writeModuleCommand.Before = applyCompressionInCommand
 
+	//
+	// Write Bootstrap artifact
+	//
+	writeBootstrapArtifactCommand := cli.Command{
+		Name:   "bootstrap-artifact",
+		Action: writeBootstrapArtifact,
+		Usage:  "Writes Mender bootstrap artifact containing no payload",
+	}
+
+	writeBootstrapArtifactCommand.CustomHelpTemplate = CustomSubcommandHelpTemplate
+
+	writeBootstrapArtifactCommand.Flags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name: "device-type, t",
+			Usage: "Type of device(s) supported by the Artifact. You can specify multiple " +
+				"compatible devices providing this parameter multiple times.",
+			Required: true,
+		},
+		artifactName,
+		cli.StringFlag{
+			Name:  "output-path, o",
+			Usage: "Full path to output artifact file.",
+		},
+		cli.IntFlag{
+			Name:  "version, v",
+			Usage: "Version of the artifact.",
+			Value: LatestFormatVersion,
+		},
+		cli.BoolFlag{
+			Name:  "no-progress",
+			Usage: "Suppress the progressbar output",
+		},
+		compressionFlag,
+		clearsArtifactProvides,
+		noDefaultClearsArtifactProvides,
+		payloadProvides,
+		payloadDepends,
+		/////////////////////////
+		// Version 3 specifics.//
+		/////////////////////////
+		artifactNameDepends,
+		artifactProvidesGroup,
+		artifactDependsGroups,
```

Comment:
* The `--no-default` options should not be here (see also my other comment about default provides).
* `--key` argument needs to be here.

Reply:
I removed the `--no-default` options, but I'm not sure which `--key` argument needs to be added  - can you please specify  it exactly?

Reply:
Done (both things)!

Reply:
Perfect, looks great!